packages/paywall/src/networkConfigs.ts

```diff
@@ -1,5 +1,5 @@
 import networks from '@unlock-protocol/networks'
-import { NetworkConfigs } from '../unlockTypes'
+import { NetworkConfigs } from '@unlock-protocol/types'
```

Comment:
That's really cool BTW! I really wanted to have repo-wide types for a looong time!

Reply:
Yes! Next step is to publish as '@types/unlock-protocol' ;)